src/link/Elf.zig

```diff
@@ -1364,6 +1365,10 @@ fn linkWithLLD(self: *Elf, comp: *Compilation) !void {
     if (self.base.options.eh_frame_hdr) {
         try argv.append("--eh-frame-hdr");
     }
+    
+    if (self.base.options.emit_relocs) {
+        try argv.append("-emit-relocs");
```

Comment:
Double slash here according to the [man page](https://manpages.debian.org/experimental/lld-10/ld.lld-10.1.en.html)

Reply:
Fixed this and the other issue in the newest commit